### DISCUSSION TITLE
Download datasets

### DISCUSSION THREAD

**Discussion by d4n227** [dmlguq456/SepReformer#4] [General]
Hi,

I'm interested for your project and I'm about to run/test it locally.
Can you please provide download links of the datasets you have used (or just for some of them)

Thank you in advance.

PS: I'm interested for audio speech separation but I'm new to this topic also not very familiar with the datasets you have used (not sure where to find them)

**Comment by dmlguq456** [comment]
I apologize for the delayed response, and thank you for your patience.

For open datasets, I recommend using LibriMix, which is based on the Librispeech dataset. While the shared code currently references a different training dataset by name, the same training process can be applied to LibriMix, from creating the .scp files to training and evaluation.

The dataset I mentioned can be easily found through a simple search.

P.S. you can find sample audio files on the demo page linked in the README. Feel free to check them out there.
